dynamic_rest/viewsets.py

```diff
@@ -337,4 +337,19 @@ def list_related(self, request, pk=None, field_name=None):
 
 
 class DynamicModelViewSet(WithDynamicViewSetMixin, viewsets.ModelViewSet):
-    pass
+
+    ENABLE_BULK_CREATION = dynamic_settings.get('ENABLE_BULK_CREATION', True)
```

Comment:
Use the special settings wrapper instead: `from dynamic_rest.conf import settings`